chromadb/api/__init__.py

```diff
@@ -550,6 +553,7 @@ def list_collections(
     def create_collection(
         self,
         name: str,
+        configuration: Optional[CollectionConfiguration],
```

Comment:
So just to confirm, the Server does not except None, only the client does and the assumption is the client will send the default?